code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -939,6 +939,15 @@ to_chat will check for valid clients itself already so no need to double check f
 		if(length(possible_silos))
 			return attempt_to_spawn_larva_in_silo(xeno_candidate, possible_silos, larva_already_reserved)
 		if(SSticker.mode?.flags_round_type & MODE_SILO_RESPAWN && !SSsilo.can_fire) // Distress mode & prior to shutters opening, so let the queue bypass silos if needed
+			if(XENODEATHTIME_CHECK(xeno_candidate))
+				if(check_other_rights(xeno_candidate.client, R_ADMIN, FALSE))
+					if(tgui_alert(xeno_candidate, "You wouldn't normally qualify for this respawn. Are you sure you want to bypass it with your admin powers?", "Bypass Respawn", list("Yes", "No")) != "Yes")
+						log_admin("[key_name(xeno_candidate)] used his admin power to bypass respawn before his timer was over")
```

Comment:
Oh actually I noticed this is wrong. This message will get sent if they press anything but yes, which is... not ideal because then it'd just be a lie. They declined to skip the queue, why is it being sent?